internal/store/model/source.go

```diff
@@ -21,6 +21,7 @@ type Source struct {
 	SshKey     string
 	Inventory  *JSONField[api.Inventory] `gorm:"type:jsonb"`
 	CredUrl    *string
+	Agents     []Agent `gorm:"constraint:OnDelete:SET NULL;"`
```

Comment:
What is now the relation between agents statuses and source status?

Reply:
good question. At this stage, the list of source statuses is the same with the agent's. Definitely, we should improve that.